packages/react/src/components/Button/BaseButton.tsx

```diff
@@ -66,6 +67,10 @@ export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonSta
     split: false,
   };
 
+  // needed to access registeredProviders when manually setting focus visibility
+  public static contextType = FocusRectsContext;
```

Comment:
Ah yeah, it's because Button is still a class component, while ChoiceGroup is a function component. This is the class-based way to consume context that I'm aware of, but let me know if you know another way.

Reply:
You can also assign this after the component by doing `BaseButton.contextType = FocusRectsContext`, which makes it easier to understand what's really happening. I'll leave it up to you if you want to make this move though.

Reply:
We have only two other `packages/react` class components that use `contextType` -- Sticky and WithResponsiveMode, and both of those use the `public static contextType` pattern, which is why I went with that one. Interestingly, Sticky doesn't explicitly provide the type for it 🙈